install_config/install/deploy_router.adoc

```diff
@@ -350,35 +350,33 @@ named `www.example.test`, see the example shown below:
       -signkey example-test.key -out example-test.crt
 ----
 
-Generate a route configuration file using the above certificate and key.
-Make sure to replace servicename `my-service` with the name of your service.
+Generate a route using the above certificate and key.
 
 ----
-# servicename="my-service"
-# echo "
+$ oc create route edge --service=my-service \ 
+    --hostname=www.example.test \
+    --key=example-test.key --cert=example-test.crt
+route "my-service" created
+$ oc get route/secured-edge-route -o yaml
```

Comment:
@kargakis Should `secured-edge-route` instead be `my-service` in this example? It seems the route name is just inherited from the service name?

Also, should we use `oc export route/my-service` instead? Looks like basically the same output `get` with `-o yaml`, just missing a few lines of metadata (AFAICT: namespace, resourceVersion, selfLink, uid).